runtime/common/src/paras_sudo_wrapper.rs

```diff
@@ -107,7 +107,7 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Downgrade a parachain to a parathread
+		/// Downgrade a lease holding parachain to an on-demand parachain
```

Comment:
They are in a way. You are competing for every block, no guarantees of price not even that you get a block when you need it. Constant block times are completely off the table.

Reply:
Sure but the block that you get is of the same quality as any other block. Just need to make sure that is clear.

Reply:
We'll eventually need to replace "upgrade" and "downgrade" when exotic scheduling comes out. I expect something like "start lease" and "end lease" would be better, since conceptually it leaves room for multiple overlapping leases or lease combined with on-demand.